index.html

```diff
@@ -190,16 +190,19 @@ <h2>Contact me</h2>
         that you need built or a project that needs coding. I’d love to help
         with it
       </p>
-      <input type="text" value="Yeremias" />
-      <input type="text" value="natajayanj14@gmail.com" />
-      <textarea
-        name="message"
-        id="message"
-        cols="30"
-        rows="10"
-        placeholder="Write your message here"
-      ></textarea>
-      <button type="submit">Get In Touch</button>
+      <form action="https://formspree.io/f/moqrvgzg" method="POST">
+        <input type="text" placeholder="Yeremias" maxlength="30" required />
+        <input placeholder="natajayanj14@gmail.com" type="email" required />
+        <textarea
+          name="message"
+          id="message"
+          cols="30"
+          rows="10"
+          maxlength="500"
+          placeholder="Write your message here"
+        ></textarea>
```

Comment:
- [ ] Good job validating the name and email fields 👍 but it would also be nice to validate our message field to make sure that the user doesn't submit an empty message. You can do that by adding `required` to your `textarea` tag.